docs/reference/dlm/apis/put-lifecycle.asciidoc

```diff
@@ -0,0 +1,85 @@
+[[dlm-put-lifecycle]]
+=== Set the lifecycle of a data stream
+++++
+<titleabbrev>Put Data Stream Lifecycle</titleabbrev>
+++++
+
+experimental::[]
+
+Sets the lifecycle of the targeted data streams.
+
+[[dlm-put-lifecycle-request]]
+==== {api-request-title}
+
+`PUT _data_stream/_lifecycle`
+`PUT _data_stream/<data-stream>/_lifecycle`
+
+[[dlm-put-lifecycle-prereqs]]
+==== {api-prereq-title}
+
+// TODO Update security
+// * If the {es} {security-features} are enabled, you must have the <role>
+// cluster privilege to use this API. For more information, see
+// <<security-privileges>>.
```

Comment:
We can drop these until we have the security model

Reply:
Done